lib/kuiq/model/process.rb

```diff
@@ -0,0 +1,29 @@
+module Kuiq
+  module Model
+    class ProcessSet
+      def initialize
+        @ps = Sidekiq::ProcessSet.new
+      end
+
+      def each
+        @ps.each do |hash|
+          yield Kuiq::Process.new(hash)
+        end
+      end
+
+      def method_missing(...)
+        @ps.send(...)
+      end
+    end
```

Comment:
Ruby requires that any implementation of `method_missing` is also accompanied by `respond_to_missing?(method_name, include_private = false)`. There is an example of that in `Kuiq::Model::Job`.

In this case, it would be:

```ruby
      def respond_to_missing?(method_name, include_private = false)
        super || @sidekiq_process_set.respond_to?(method_name, include_private)
      end
```